### ISSUE TITLE
`apropos` can't search in Aka aliases

### ISSUE THREAD

**Issue by nyuszika7h** [ProgVal/Limnoria#1081]
```
17:28:20 <+nyuszika7h> +apropos ipv6
17:28:21 <@Caroline> No appropriate commands were found.
17:29:46 <+nyuszika7h> +help ipv6subnet
17:29:47 <@Caroline> (ipv6subnet <an alias, 1 argument>) -- Alias for "calc 2**(128-$1)".
```

Maybe we should add an `aka search` command?

**Comment by Mikaela** [comment]
> Maybe we should add an aka search command?

In my opinion Apropos should be able to detect aliases too.

**Comment by nyuszika7h** [comment]
The reason this issue exists is probably the same as the reason `list Aka` no longer lists the aliases (which is a good thing).

**Comment by GLolol** [comment]
Aka overrides the listCommands directive, to prevent aliases from showing up in `list Aka` along with regular commands. I don't think adding a special case for Aka in apropos is appropriate, so I implemented an `aka search` conmmand (#1097) instead.
